dstep/translator/MacroDefinition.d

```diff
@@ -2053,6 +2053,10 @@ bool translateFunctAlias(
     {
         Identifier ident = cast(Identifier) expr.expr;
 
+        import dstep.core.Traits : isKeyword;
+        if (ident.spelling.isKeyword())
+            return false;
```

Comment:
This case is a bit different - you simply can't alias a keyword. For example `alias foo = assert` is not legal D code, no matter what you do.